packages/cardano-services/test/cli.test.ts

```diff
@@ -2671,6 +2671,37 @@ describe('CLI', () => {
         done();
       });
     });
+
+    it('exits with code 1 with an invalid SCHEDULES path', (done) => {
+      expect.assertions(2);
+      proc = withLogging(
+        fork(
+          exePath,
+          [
+            ...commonArgs,
+            '--queues',
+            'pool-metadata',
+            '--postgres-connection-string-db-sync',
+            postgresConnectionString,
+            '--postgres-connection-string-stake-pool',
+            postgresConnectionStringStakePool,
+            '--stake-pool-provider-url',
+            'http://localhost:4000/stake-pool',
+            '--schedules',
+            'does_not_exist'
```

Comment:
I'm not convinced on using file to specify schedules, I think it makes it more difficult to use, as all of the other options are just direct arguments. Let's consider `--pool-delist-schedule '0 * * * *'`. Thoughts @ginnun @iccicci ?

Reply:
Personally I don't like so much combining configurations through files and environment variables, but I understand and agree your point @ginnun .

I would vote to keep this as is, once we have some other idea we can improve this.

Reply:
In my opinion for the rest of the configurations, we should switch to simple key/value program config files, which could be overridden by an environment or CLI param.